lib/drivers/dexcomDriver.js

```diff
@@ -857,9 +857,11 @@ module.exports = function (config) {
 
 
   return {
-    // using the default detect for this driver
-    // detect: function(cb) {
-    // },
+    //bypassing the detect function for now
+    detect: function (obj, cb) {
+      debug('Dexcom Detect!');
```

Comment:
If we're still doing this bypass even after a more thorough dig into the El Capitan changes, maybe it'd be better to have the debug message 'Bypassing Dexcom detect' or something, seems a little odd to me to have added a debug message for what's basically a no-op ;) But I'm gonna merge so Howard can test the build.